src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const array = sourceString.split(';');
```

Comment:
[CODE STYLE] Avoid variable namings like `array`. It's better to give your variables more descriptive names that reflect their contents or purpose. For example, `styleEntries` would be a more descriptive name in this case.